archetype-common/pom.xml

```diff
@@ -138,6 +169,12 @@
       <artifactId>junit</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.hamcrest</groupId>
+      <artifactId>hamcrest-core</artifactId>
+      <version>2.2</version>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Instead of using it you can exclude `hamcrest-core` from junit and then use `hamcrest`

Reply:
nice, done